src/ProjectTemplates/Web.ProjectTemplates/content/StarterWeb-CSharp/Startup.cs

```diff
@@ -4,20 +4,10 @@
 using System.Threading.Tasks;
 #if (OrganizationalAuth || IndividualB2CAuth)
 using Microsoft.AspNetCore.Authentication;
-#endif
-#if (OrganizationalAuth)
-using Microsoft.Identity.Web;
-using Microsoft.Identity.Web.UI;
-using Microsoft.Identity.Web.TokenCacheProviders.InMemory;
-#if (MultiOrgAuth)
 using Microsoft.AspNetCore.Authentication.OpenIdConnect;
-#endif
-using Microsoft.AspNetCore.Authorization;
-#endif
-#if (IndividualB2CAuth)
 using Microsoft.Identity.Web;
 using Microsoft.Identity.Web.UI;
-using Microsoft.Identity.Web.TokenCacheProviders.InMemory;
+using Microsoft.AspNetCore.Authorization;
```

Comment:
sort

Reply:
Fixed

Reply:
Hmm seems like it isn't sorted, move this line to line 8